incubator/kiam/templates/server-daemonset.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: extensions/v1beta1
```

Comment:
apps/v1beta2

Reply:
This caused the addition of

```yaml
  selector:
    matchLabels:
```

to `spec`.